src/Store/configureStore.js

```diff
@@ -1,9 +1,29 @@
-import { createStore } from 'redux';
+import { AsyncStorage } from 'react-native';
+import { createStore, applyMiddleware } from 'redux';
+import { persistStore, persistReducer } from 'redux-persist';
+import { createLogger } from 'redux-logger';
+
 import rootReducer from './Reducers';
 import fixtures from '../Fixtures';
 
-export default createStore(
-  rootReducer,
+const persistConfig = {
+  key: 'root',
+  storage: AsyncStorage,
+  whitelist: ['completeTrainings', 'favoriteDrills'],
+  blacklist: ['drills', 'trainings', 'programs'],
+};
+// Middleware: Redux Persist Persisted Reducer
+const persistedReducer = persistReducer(persistConfig, rootReducer);
+
+// Redux: Store
+const store = createStore(
+  persistedReducer,
   fixtures,
   window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+  applyMiddleware(createLogger()),
```

Comment:
Maybe we should add this only on development environment ?

Reply:
Ok I tried, using composeWithDevTools with does this if I understood the doc correctly.

Reply:
As it is now, this line does nothing I think. `createStore` only takes 3 parameters.

Reply:
yes, forgot to push :)